packages/intuit/configuration-schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "baseUrl": {
+      "title": "Base URL",
+      "type": "string",
+      "description": "The Quickbooks(intuit) base URL",
```

Comment:
Should we default this? Can users even us different domains?

If our implementation is on the main intuit.com baseUrl, we should default it and leave this as optional